src/nvim/mbyte.c

```diff
@@ -101,56 +101,14 @@
 #include "nvim/os/os.h"
 #include "nvim/arabic.h"
 
-typedef struct {
-  int rangeStart;
-  int rangeEnd;
-  int step;
-  int offset;
-} convertStruct;
-
-struct interval {
-  long first;
-  long last;
-};
-
 #ifdef INCLUDE_GENERATED_DECLARATIONS
 # include "mbyte.c.generated.h"
+# include "mbyte/dbcs.h.generated.h"
+# include "mbyte/latin.h.generated.h"
+# include "mbyte/utf8.h.generated.h"
```

Comment:
You should include `mbyte/dbcs.h` (and others) above with regular includes and include `.h.generated.h` files *there* and not here.